configuration/upgrade-SQL-to-managed-identity-authentication.md

```diff
@@ -0,0 +1,126 @@
+   * With this, you will gain permission to perform changes in the key vault
+
+![Key Vault - Access Policy - Principal](../.gitbook/assets/upgrade-SQL-to-managed-identity-authentication_key-vault-create-access-policy-principal.png)
+
+* The Application section can be skipped, so **select Next**
+* Under the **Review + Create** section, check to make sure all the information is correct and **click Create (8)**
+  * This will complete the process
+
+![Key Vault - Access Policy - Review and Create](../.gitbook/assets/upgrade-SQL-to-managed-identity-authentication_key-vault-create-access-policy-review-and-create.png)
+
+When this is set up, **the Access Policies screen should now show the name of the person selected** during the creation process under the **User** section. 
+
+If this was completed successfully, proceed to the following steps:
+
+* On the Key Vault resource, **select Networking (1)**
+   * Located on the left side under Settings
+* Select **Allow public access from specific virtual networks and public IP addresses (2)**
+* Under Firewall, **click Add your client IP address** and **paste your public IP address (3)** here
+* **Click Apply** when finished
+
+![Key Vault - Networking](../.gitbook/assets/upgrade-SQL-to-managed-identity-authentication_key-vault-networking.png)
+
+* **Select Secrets** (1)
+  * Located on the left side under Objects
+* **Select the PointDatabase option (2)** from the list of available names
+
+![Key Vault - Secrets](../.gitbook/assets/upgrade-SQL-to-managed-identity-authentication_key-vault-db-secret.png)
+
+* On the PointDatabase screen, **click the New Version button (3)**
+
+![Key Vault - Secrets - Point Database](../.gitbook/assets/upgrade-SQL-to-managed-identity-authentication_key-vault-db-secret-new-version.png)
+
+* On the form that opens, you will need to enter the following text as **Secret value (4)**
+
+ `Data Source=tcp:{serverName},1433;Initial Catalog=SysKitPointDB;User Id="{managed identity client id}";Authentication=Active Directory Managed Identity;`
+
+{% hint style="warning" %}
+**IMPORTANT!**  
+There are two parts of the above text that will need to be changed:
+
+Data Source=tcp:**{serverName}** should be pasted from your SQL server for SysKit Point. The text is formatted as: servername.database.windows.net
+
+User Id="**{managed identity client id}** needs to be changed. The Client ID you previously copied from your Managed Identity resource should be placed where the text says **managed identity client id**. 
+{% endhint %}
+
+* When the **edited text** has been entered, **click Create**
+
+![Key Vault - Create a secret](../.gitbook/assets/upgrade-SQL-to-managed-identity-authentication_key-vault-db-secret-create.png)
+
+
+After successfully setting this up, the next step will be removing the access granted to perform the previous changes to secure SysKit Point Azure resources. 
+
+* **Under Networking, remove your IP address**
```

Comment:
Fali, set Public network access to Disabled
![image](https://user-images.githubusercontent.com/12006653/195590910-214bb66e-fdd4-4496-a55f-95a5d8c55cc0.png)

Reply:
nope, krivi resurs,s ry